Added more docs for query search

marciogualtieri-polyswarm opened the pull request:
Adding more examples to readme files on query searches.

mrtizmoatwork commented on README.md:
````diff
@@ -201,6 +292,49 @@ File 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267
 ```
 
+```bash
+$ polyswarm_api -o /tmp/test.txt search search metadata '{ "query": { "exists": {"field": "lief.libraries"}}}'
````
should there be 2 "search" words in a row?

marciogualtieri-polyswarm replied:
I don't know how I put this in there. Will fix.

mrtizmoatwork commented on README.md:
```diff
@@ -201,6 +292,49 @@ File 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267
+	SHA1: 29629198ac945de358952337da1015a8252c11f1
+	MD5: e74b9320e6c15363fa30319af75ff3e5
+	First seen: Thu, 16 May 2019 14:26:26 GMT
+	Observed countries: ,PR
```
the output of this value has a problem with the first comma ",". It should be omitted when there is only one value.

marciogualtieri-polyswarm replied:
Will investigate.

marciogualtieri-polyswarm replied:
Fixed a bug in the output.

marciogualtieri-polyswarm commented:
> the documentation in README.md looks good except for 2 minor things. Please also add the relevant info to API.md for the updated search endpoint API.

Thanks for the review, Nick. I fixed the typo, the bug in the output, and updated API.md.

mrtizmoatwork commented:
@supernothing do you have any additional documentation that you want to add? Could add it to this PR.

marciogualtieri-polyswarm commented:
> looks good to me now.

Thanks, Nick. Do you want a review from @supernothing as well or should I merge it now?

marciogualtieri-polyswarm commented:
> @supernothing do you have any additional documentation that you want to add? Could add it to this PR.

Cool, will wait until Ben have a look.

supernothing commented:
I re-arranged some stuff that I had been meaning to, but otherwise looks good. @mrtizmoatwork final OK?
